pootle/apps/pootle_store/unit/timeline.py

```diff
@@ -60,24 +62,46 @@ def gravatar_url(self, size=80):
 
 class SuggestionEvent(object):
 
-    def __init__(self, submission_type, username, full_name):
+    def __init__(self, submission_type, username, full_name, comment):
         self.submission_type = submission_type
         self.username = username
         self.full_name = full_name
+        self.comment = comment
 
     @cached_property
     def user(self):
         return DisplayUser(self.username, self.full_name)
 
     @property
     def description(self):
-        author_link = self.user.author_link
+        params = {
+            'author': self.user.author_link
+        }
+
+        if self.comment:
+            params.update({
+                'comment': format_html(u'<span class="comment">{}</span>',
+                                       self.comment),
+            })
+
         description_dict = {
             SubmissionTypes.SUGG_ADD: _(u'Added suggestion'),
-            SubmissionTypes.SUGG_ACCEPT: _(u'Accepted suggestion from %s',
-                                           author_link),
-            SubmissionTypes.SUGG_REJECT: _(u'Rejected suggestion from %s',
-                                           author_link)}
+            SubmissionTypes.SUGG_ACCEPT: _(
```

Comment:
can we break these out to separate methods/attributes/properties

they are kinda hard to follow atm